packages/tools/src/utilities/planar/filterAnnotationsForDisplay.ts

```diff
@@ -42,7 +42,10 @@ export default function filterAnnotationsForDisplay(
         return false;
       }
 
-      const imageId = annotation.metadata.referencedImageId;
+      const imageId = annotation.metadata.referencedImageId?.replace(
```

Comment:
I didn't expect this, why?

Reply:
Added a comment, but basically we should determine whether or not to display given the targetId, as that is actually specifying where it gets displayed.  I can remove this for now if you prefer, and simply remove the imageId prefix in the interpolation code for now, and we can discuss how to really handle referencedImageId/targetId differences.